### ISSUE TITLE
default CES parameter files for buildings/simple miss enhgab

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#380]
13 out of 29 CES parameter files for `buildings/simple` miss entries for `enhgab`
```
/p/projects/remind/inputdata/CESparametersAndGDX (master)$ grep -c enhgab *buil_simple*.inc | grep ":0$"
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SDP-GDP_gdp_SDP-Kap_perfect-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP1-GDP_gdp_SSP1-Kap_perfect-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-EDGET-FE_med-Reg_2b1450bc.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_imperfect-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_771e2a8eff.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_b5763db2f6.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP5-GDP_gdp_SSP5-Kap_perfect-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_wise-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-demTrsp_conv-Reg_62eff8f7.inc:0
stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-demTrsp_wise-Reg_62eff8f7.inc:0
stat_off-indu_subsectors-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_62eff8f7.inc:0
```
which crashes REMIND with `buildings/simple`.
https://github.com/remindmodel/remind/blob/8731a7091866060eeb046c27622c42406643785b/modules/36_buildings/simple/sets.gms#L28-L34
Question is: how where these files generated?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
All `Kap_perfect` seem affected, but also some `Kap_debt_limit`.
  |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SDP | GDP_gdp_SDP | Kap_perfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP1 | GDP_gdp_SSP1 | Kap_perfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | EDGET | FE_med | Reg_2b1450bc | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_imperfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect |   |   | Reg_771e2a8eff | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect |   |   | Reg_b5763db2f6 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP5 | GDP_gdp_SSP5 | Kap_perfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_wise |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect | demTrsp_conv |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect | demTrsp_wise |   | Reg_62eff8f7 | 0
stat_off | indu_subsectors | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_perfect |   |   | Reg_62eff8f7 | 0
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SDP | GDP_gdp_SDP | Kap_debt_limit |   |   | Reg_62eff8f7 | 1368
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP1 | GDP_gdp_SSP1 | Kap_debt_limit |   |   | Reg_62eff8f7 | 1368
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit |   |   | Reg_62eff8f7 | 1368
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP5 | GDP_gdp_SSP5 | Kap_debt_limit |   |   | Reg_62eff8f7 | 1368
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | FE_med |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | FE_med_lowElElast |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | INNO_FE_low |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | INNO_FE_med |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | INNO_FE_med_highElast |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | INNO_FE_med_lowElast |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_complex | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit |   |   | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv | FE_med | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv | FE_med_ariad_highElast | Reg_c1294fae9c | 2394
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv | FE_med_ariad_lowElast | Reg_c1294fae9c | 2394
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv | FE_med_lowElElast | Reg_2b1450bc | 2394
stat_off | indu_fixed_shares | buil_simple | tran_edge_esm | POP_pop_SSP2 | GDP_gdp_SSP2 | Kap_debt_limit | demTrsp_conv |   | Reg_2b1450bc | 2394

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Last commits to modify these files 
```
$ grep -lv /p/projects/remind/inputdata/CESparametersAndGDX/*buil_simple*.inc | sed 's|/p/projects/remind/inputdata/CESparametersAndGDX/|./modules/29_CES_parameters/load/input/|' | xargs -n 1 git rev-list -n 2 HEAD -- | awk '!a[$0]++' | tail -n +2 | xargs -n 1 git log -n 1 | cat
commit 746b5a52460f2c1dfd944c6f66f1744bbbe4830e
Author: Lavinia Baumstark <baumstark@pik-potsdam.de>
Date:   Thu Mar 25 15:22:05 2021 +0100

    adapt to changes in madrat using hash "xxxhash32" instead of "md5", delet CES parameters for old regionmappingREMIND
commit 4ac0b6e5d6ce58e216a931370bd7b8253fc298f7
Author: Lavinia Baumstark <baumstark@pik-potsdam.de>
Date:   Mon Dec 9 12:07:59 2019 +0100

    *bugfix* added missing files
commit 3e188027fd2bc29bf2c09ec368be0f9815bd4a47
Author: Marianna Rottoli <marianna.rottoli@mail.polimi.it>
Date:   Thu Mar 25 15:08:12 2021 +0100

    Calibration files for EDGE-T ariadne REMIND-EU.
```
All but one file have the same MD5 checksum in the CESparametersAndGDX repository as when they were deleted from the REMIND repository.
| commit  | unmodified | file |
|---------|------------|------|
| 3e18802 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-FE_med_ariad_highElast-Reg_c1294fae9c.inc |
| 3e18802 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-FE_med_ariad_lowElast-Reg_c1294fae9c.inc |
| 4ac0b6e | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_771e2a8eff.inc |
| 4ac0b6e | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_b5763db2f6.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SDP-GDP_gdp_SDP-Kap_perfect-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP1-GDP_gdp_SSP1-Kap_debt_limit-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP1-GDP_gdp_SSP1-Kap_perfect-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-EDGET-FE_med-Reg_2b1450bc.inc |
| 746b5a5 | ✗ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-FE_med-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-FE_med_lowElElast-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-INNO_FE_low-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-INNO_FE_med-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-INNO_FE_med_highElast-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-INNO_FE_med_lowElast-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_imperfect-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP5-GDP_gdp_SSP5-Kap_debt_limit-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP5-GDP_gdp_SSP5-Kap_perfect-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-FE_med-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-FE_med_lowElElast-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-Reg_2b1450bc.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_conv-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_debt_limit-demTrsp_wise-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-demTrsp_conv-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_fixed_shares-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-demTrsp_wise-Reg_62eff8f7.inc |
| 746b5a5 | ✓ | stat_off-indu_subsectors-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-Kap_perfect-Reg_62eff8f7.inc |

**Comment by Renato-Rodrigues** [comment]
New calibrations are necessary for all missing combinations. 

When I introduced the H2 split, I gave a warn about that in mattermost, by email and in the remind meeting. Only the calibrations for the default module realizations were carried at the time, and all others should be updated as needed.

This affects runs using the buildings simple and industry fixed shares realizations.

PS: Soon we will have another CES structure change in the buildings simple realization that will require new calibrations also (the split of electricity use into conventional, heat pumps and resistive heating). So I would wait for that change to be integrated before spending much time in creating new calibrations if they are not urgent.
